client/client-multi/client-c/src/kaa/platform-impl/posix/key_utils.c

```diff
@@ -20,13 +20,42 @@
 #include <sys/stat.h>
 #include <string.h>
 #include <platform/ext_key_utils.h>
+#include <mbedtls/pk.h>
+#include <mbedtls/entropy.h>
+#include <mbedtls/ctr_drbg.h>
+#include <mbedtls/md.h>
+
+#define KAA_SESSION_KEY_LENGTH         16
```

Comment:
I think some of these definitions can be moved in the source file (instead of keeping it in header). Consider which you can move there.